ruby/webapp.rb.in

```diff
@@ -1,5 +1,20 @@
 plugin_path = "@CMAKE_SOURCE_DIR@/public" 
 
+#load dirs containing scripts
```

Comment:
I agree with Steffen here. Its neither a c++ nor a ruby package. Yes, using cmake to just install one file is a bit convoluted, but would be the same using rake for it (and I don't know rake that well).